package.json

```diff
@@ -220,11 +220,11 @@
   "proxyURL": "http://localhost:8000",
   "scripts": {
     "test-wrap": "node scripts/test.js",
```

Comment:
Can we kill this then?

Reply:
If no-one uses it, yes. Maybe some engineer does?

Reply:
almost for sure not

Reply:
removed it in a new commit @evanpurkhiser